v2/contribute/decisions/thirdparty/0002-combine-static-and-dynamic-provider-list.mdx

```diff
@@ -0,0 +1,193 @@
+---
+id: "0002"
+title: Combining statically and dynamically configured provider list
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Combining statically and dynamically configured provider list
+
+<DecisionHeader status="proposed" lastUpdate="2022-12-08" created="2022-12-08" deciders={["rishabhpoddar", "sattvikc"]} proposedBy={["sattvikc"]} />
+
+## Context and Problem Statement
+
+Since we now allow users to dynamically add provider configurations into the core, and also the user might have configured a static list, this causes an ambiguity on what should be the final set of providers list and which configs should be used by the SDK
+
+How do we come up with a consolidated list of providers and their configs?
+
+## Considered Options
+
+* Prioritize list from the core, merge the config on to the static list prioritizing the config from core. (Refer the option section for more details)
```

Comment:
there were more options we did consider - like if defined in the core, we should just use that and not merge with static

Reply:
Done, added that